crawl-ref/source/dat/descript/spells.txt

```diff
@@ -141,11 +141,36 @@ deplete magical power.
 %%%%
 Blade Hands spell
 
-Causes long, scythe-shaped blades to grow from the caster's hands, increasing
-melee damage significantly.
+{{
+    if you.hand() == "tentacle" then
+        return "Causes long, scythe-shaped blades to grow from the " ..
+               "caster's tentacles, increasing melee damage significantly." ..
+               "\n\nWhile transformed, equipped weapons, shields, and " ..
+               "gloves are melded, and casting spells becomes somewhat " ..
+               "more difficult."
+    elseif you.hand() == "paw" then
+        return "Causes long, scythe-shaped blades to grow from the " ..
+               "caster's front paws, increasing melee damage significantly." ..
+               "\n\nWhile transformed, equipped weapons, shields, and " ..
+               "gloves are melded, and casting spells becomes somewhat " ..
+               "more difficult. In addition, the clacking of your blades " ..
+               "on the ground makes you slightly less stealthy."
+    elseif you.hand() == "claw" then
+        return "Causes long, scythe-shaped blades to grow from the " ..
+               "caster's claws, increasing melee damage significantly.\n\n" ..
+               "While transformed, equipped weapons, shields, and gloves " ..
+               "are melded, and casting spells becomes somewhat more " ..
+               "difficult."
+    else
+        return "Causes long, scythe-shaped blades to grow from the " ..
+               "caster's hands, increasing melee damage significantly.\n\n" ..
+               "While transformed, equipped weapons, shields, and gloves " ..
+               "are melded, and casting spells becomes somewhat more " ..
+               "difficult."
+    end
+}}
```

Comment:
```suggestion
{{
    local hands = you.hand() .. "s"

    return "Causes long, scythe-shaped blades to grow from the " ..
           "caster's " .. hands .. ", increasing melee damage significantly." ..
           "\n\nWhile transformed, equipped weapons, shields, and " ..
           "gloves are melded, and casting spells becomes somewhat " ..
           "more difficult."
}}
```

No need for a case statement. If a species with an irregular-plural hand noun gets added this will break, but its better to minimize duplication.

Reply:
Thanks for the suggestion. I got rid of the duplication but left a special case for paws because I wanted to include the info about Felids having reduced stealth while using Blade Hands.